src/controllers/donorController.ts

```diff
@@ -171,11 +173,19 @@ export const modifyDonation = (req: Request, res: Response) => {
 export const userDonations = (req: Request, res: Response) => {
     const id = req.params.donor_id;
     return Donation.find({ donor: id })
-        .then(result => res.status(200).json({ success: true, donations: result }))
+        .then(result => res.status(200).json({ success: true, donation: result }))
```

Comment:
why change `donations` to `donation`?

Reply:
Sorry about that—it should be donations. Since I was working on both tasks during sprint day I got this confused with my getDonationDetails code